target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4019-eap2200.dts

```diff
@@ -0,0 +1,410 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/clock/qcom,gcc-ipq4019.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+
+/ {
+	model = "EnGenius EAP2200";
+	compatible = "engenius,eap2200";
+
+	chosen {
+		bootargs-append = " root=/dev/ubiblock0_1";
+	};
+
+        memory {
+                device_type = "memory";
+                reg = <0x80000000 0x10000000>;
```

Comment:
Not needed, Linux can identify the size by itself

Reply:
Thank you for your patience